src/styles/main.scss

```diff
@@ -0,0 +1,194 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+    display: block;
+    text-decoration: none;
+  }
+
+  .nav {
```

Comment:
Each separate BEM block should be in a separate file, same about all the block below